src/objects.rs

```diff
@@ -134,8 +134,33 @@ where
 
 /// Trait covering all objects with an id.
 pub unsafe trait HasId: RoomObjectProperties {
-    fn id(&self) -> String {
-        js_unwrap!(@{self.as_ref()}.id)
+    /// Retrieves this object's id as an untyped, packed value.
+    ///
+    /// This has no major differences from [`HasId::id`] besides the return
```

Comment:
besides -> beside?

Reply:
Right - I think I almost never use `besides` correctly...

Reply:
Meh, I'm not married to either, hence the question mark. Feel free to leave it like that.